demo/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+apply from:  rootProject.file("$rootDir/common.gradle")
+
+android {
+    namespace 'com.uid2.demo'
```

Comment:
@IanDBird and I already talked about this but the name of this app is going to be changed to `DevApp` instead of `DemoApp`.  The reason is that this app's primary purpose is to serve as a development tool for building the SDK and not an officially recommended demonstration of how the SDK works for customers.  We'll likely have more polished examples for that purpose down the road.

Reply:
Yup, will make the change as part of this.